### DISCUSSION TITLE
Governing Board Candidates 2023

### DISCUSSION THREAD

**Discussion by wipfli** [maplibre/maplibre#299] [Governing Board Election]
The Governing Board elections will take place online on Thursday, August 24th, 2023.

Candidates for the Governing Board can register themselves up to two weeks before the election, i.e., until **Thursday, August 10th, 2023** by posting in this thread that they are interested in serving on the Board. Please include a description of your involvement in MapLibre and disclose if you are paid for working on MapLibre or have a vested interested in the Project.

Each nomination should be a separate comment. Use the "reply" section under each nomination to ask questions.

**Comment by HarelM** [comment]
Hi, my name is Harel Mazor and I would like to nominate myself for the governing board of MapLibre.
I was elected last time and I would like continue being a board member.
For transparency, I'm also the lead web maintainer of MapLibre gl js.
I'm very much interested in the success of MapLibre because I really believe in open source, and partially since I'm using it in a project I co-founded called Israel Hiking Map.
As a board member I try my best to raise the community's voice as best as I can.
It would be my honor if you vote for me.
Thanks! 😀

**Comment by nyurik** [comment]
Dear MapLibre Community,

I'm seeking your support to continue serving on the MapLibre board of directors. As a passionate advocate for open-source mapping software, I'm committed to driving MapLibre's mission forward.

Being one of the founders of MapLibre overseeing our finance (both raising and distributing), here's what I did in the last 20 months:
* I proposed the MapLibre organization, created a GitHub org, and collaborated on rebranding and publishing the initial fork on NPM.
* Set the vision for MapLibre to become more than just a single map visualization library.

I believe MapLibre should offer a complete mapping software stack -- "from bits to pixels": given data, the stack should offer a clear path to convert it into tiles, serve tiles to the world (Martin), visualize them with MapLibre web / native / rs, and offer extra functionality with plugins as well as style editing with Maputnik. The list goes on.

To achieve this vision, I have on-boarded the Martin tile server to the MapLibre organization, and I have actively maintained it, raising it to 1,250 GitHub stars. I have attracted many new contributors. I organized next generation tile format discussion, onboarded MapLibre-rs - an experimental Rust reimplementation of MapLibre to target all platforms from a single codebase.

* I have set up a non-profit project under the OpenCollective umbrella as the primary financial assignee, being responsible for the MapLibre finances (primary signature authority)
* Presented MapLibre at numerous mapping conferences such as FOSS4G and SOTM.
* Engaged with many donors, including the initial discussions with AWS and Meta at the SOTMUS in Arizona, where I was able to secure the majority of the initial funding (huge thanks goes to the donors, and to Oliver for coordinating)

Yes, but what about next year? I believe MapLibre must have three fundamental goals:
* Grow developer community
* Maintain and improve existing code
* Research next-generation stack with latest technology (WebGPU, faster tiles, multi-platform, ...)

All three goals are important - without the community we won't be able to maintain a vast codebase. Without investment into the next generation technology, we will quickly become irrelevant.

Thank you for your support!

P.S. Financial disclosure:
* I have NOT received any payments for my MapLibre work from the organization.
* I have used some of MapLibre projects in my regular employment (e.g. Martin), but most of my efforts are unrelated to my day job.

**Comment by birkskyum** [comment]
Hi, I'm Birk.

**Background** - I've wrangled maps and geodata for 15 years, in use-cases varying from estate, maritime, agriculture, logistics, environmentally etc., and I'm a co-founder of maplibre.

**Past year** - My focus on the board has been to to get us positioned better regarding the changing landscape of graphics API's. While we're not done here, I'm very pleased with the current state of affairs:

- **Native** - We have Metal support in MapLibre Native well underway ([link](https://github.com/orgs/maplibre/projects/8/insights/2))
- **GL JS** - I championed landing the WebGL2 support in MapLibre GL JS v3, which required a significant refactor of our tests suite and shaders. A note on WebGPU - The browser support is still only at [24%](https://caniuse.com/?search=webgpu), so it's not ready for broad adoption yet, but we can start some planning and have it in mind. I made a tacking issue for us [here](https://github.com/maplibre/maplibre-gl-js/issues/2606).
- **MapLibre RS** - I initiated the move of the R&D WebGPU-compatible MapLibre RS into a supported tier ([link](https://github.com/maplibre/maplibre/pull/282)). Looking forward to see what insights it will bring.

**Future** - We got the foundation of the organization in place now, maintenance is running and graphics APIs are on the move, so over the next year I'll focus on two things: spending our funds more efficiently, and pushing new features. I've taking multiple initiatives inside the Board regarding these topics, and I believe we can make a lot of progress here in the next term.

1. Increase our pace of innovation. 
    * I'll be working on optimizing workflows, and remove time-sinks.
    * I'll be working on increasing the number of staff-dev-hours we have, as increasing our capacity to handle and create PRs will allow us to get more stuff done.

2. Deliver on the most requested features. We haven't been able to secure meaningful progress for years on some key requested features, and I'd be working on changing that over the next term. These features are related and include:
    * Loading non-mercator tiles
    * Atmosphere/fog
    * Adaptive projections
    * Globe view

This community is a pleasure to work with, so there's a lot of reasons to be excited about the future of MapLibre! I'm not paid for my work on the board, but I'd be honored to continue helping us grow.

**Comment by lseelenbinder** [comment]
Hi, I'm Luke.

I've been part of MapLibre since the beginning—helping with the initial communication, organizational setup, and a member of the governing board since inception. I hope to continue serving the community as a member of the board and active participant.

Beyond my board responsibilities, I've initiated and assisted with transitioning Maputnik to MapLibre, worked on maplibre-gl-leaflet, built a search box plugin for MapLibre, assisted with Martin and other Rust-related projects, helped ensure the continued success of the Native Metal project, and connected with potential and current sponsors (including komoot, AWS, and Meta). I also initiated the decision to create classes of projects to allow us to better allocate funds.

In the next few years, I want to see and develop a few things:

- Overhaul and improvements to the **Bounty program** to better foster _community_ and _code_ development.
- Completion and success of the **MapLibre Native Metal** project (including modernization of the codebase).
- On-going support and success of our **sponsorship program**.
- Continued support for the **MapLibre RS** project, with a view to seeing Rust code increasingly support and supplement C++ in the native codebase and eventually become the flagship project as support for WASM & WebGPU increases.
- A vibrant, diverse **community** that can increasingly develop significant features and be **self-sustaining**.
- More **dedicated developers and maintainers** (either via MapLibre hiring or sponsor's hiring) to assist the community and ensure continued high-quality code and process.

Financial & Conflict of Interest Disclosure: I participate in MapLibre as an individual and as part of my company, Stadia Maps. I have not been compensated by the organization, but I am compensated for time spent on MapLibre by my company.

**Comment by jonahadkins** [comment]
Hello MapLibre Community 👋 

My name is Jonah Adkins, and I'm thrilled to announce my candidacy for a position on the MapLibre Governing Board. I believe I would bring a distinctive viewpoint to the board, rooted in my deep involvement in map design and a keen awareness of how technological decisions impact it.

My journey into GIS technologies began in 1999, and for the past decade, I've been dedicated to the world of technical cartography. I am an active member of the OpenStreetMap foundation, OpenStreetMap US, and NACIS, where I collaborate and contribute regularly. I also have a track record of community involvement, having previously served as a board member for the OpenStreetMap US community for several years. Prior to that, I was a board member for VAMLIS, contributing to the Virginia GIS community.

Presently, I am the Technical Cartography Lead at Meta, where I work with MapLibre and associated technologies, powering a considerable volume of maps.

I am enthusiastic about guiding MapLibre's trajectory and am committed to championing the integration of thoughtful design with robust technology.

Thanks for your consideration!

**Reply by louwers** [reply]
Hello @jonahadkins, awesome to see you as a candidate. This week is 'voting week', I already know the other candidates (i.e. the current board) so I wanted to ask you a question. Could you share your vision or ideas about MapLibre's trajectory? Is there one aspect you feel is especially important or is there something you feel particularly enthusiastic about given your experience, and why?

**Reply by jonahadkins** [reply]
Hey @louwers , Thanks for question!

I would say my vision for MapLibre is not much different from others here. I want to help make it the leading library for web and mobile developers to create stunning, performant maps. Specifically, I believe it's crucial for MapLibre to be well-positioned for upcoming technologies like 3D tiles and cloud-optimized formats, leveraging our corporate partnerships to help us achieve positive outcomes that benefit the community, and finding new ways to energize the community toward the bug-bounty program.

Personally, my extensive work in web cartography over the last several years, along with my experience within similar open-source communities, has ignited my enthusiasm about a few key aspects:

- Removing barriers to crafting beautiful maps using MapLibre.
- Fostering open, respectful communication within the community and working together openly.
- Showcasing the impressive maps created by users to highlight MapLibre's capabilities.
- Exploring innovative methods to promote and spread awareness about MapLibre.

**Reply by louwers** [reply]
Thanks for your answer! 🙂

**Comment by klokan** [comment]
Hi everyone!

My name is Petr Pridal known as Klokan. I am co-founder of MapLibre, founder of OpenMapTiles, EPSG.io, author of GDAL2Tiles, architect of OldMapsOnline.org, contributor to open-source cartography on multiple fronts and multiple projects, entrepreneur and CEO of MapTiler.com.

I strongly believe in viability of open-source components for developers.

Here I offer my services to the MapLibre project and community - as self-nominee for the governing board.
If you vote for me again, I can continue to contribute with improvement of processes and governance of the project - while helping to shape the future with strategical insights from industry and with executive experience.
I have a degree in Computer Science, I have been coding myself for years, I also have extensive practical experience in management as well as education in board governance from IMD in Switzerland.

During the last year, I was fighting on the board of MapLibre for:
1) measurable goals aligned to [the project strategy](https://drive.google.com/file/d/1F5abpGx8VlVse4SHGwxgRRY1wQ29i-0_/view)
2) documented roles & responsibilities and processes (see subpages at [wiki](https://github.com/maplibre/maplibre/wiki))
3) transparent accounting (have a look yourself at https://opencollective.com/maplibre#category-BUDGET)
4) very clear structured offer on [sponsorship](https://maplibre.org/files/Slide-Deck-MapLibre-Sponsorship-Program.pdf) leading to raising money, allowing employment of maintainers in this open-source project
5) trademarks - I donated the MapLibre EU trademark to the organization, and pushed forward the registration of US trademark
6) focus on becoming "the best rendering library" out there with innovation and clear roadmap

Future? See the top voted directions I have proposed for [project roadmap](https://github.com/maplibre/maplibre/discussions/categories/bounties-cross-project-directions?discussions_q=is%3Aopen+category%3A%22Bounties+-+cross-project+directions%22+sort%3Atop+author%3Aklokan).

In previous years me and my team helped with setting up of this project, branding and identity of the project, we contributed in code on the initial version of Native, pushed initial announcements and assisted on many other fronts.
